frontend/packages/dev-console/src/components/guided-tour/GuidedTourIntroBanner.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from 'react';
+
+const GuidedTourIntroBanner: React.FC = (props) => (
```

Comment:
also, these brand graphics usually have variants for both light and dark themes. I think we should use either variant based on the current console theme